src/neuroconv/datainterfaces/behavior/lightningpose/lightningposeconverter.py

```diff
@@ -189,7 +174,6 @@ def run_conversion(
         overwrite: bool = False,
         reference_frame: str | None = None,
         confidence_definition: str | None = None,
-        external_mode: bool = True,
```

Comment:
I'd like to deprecate this parameter for six months first before removing it outright.